Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.yml

```diff
@@ -0,0 +1,468 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: AlienVault_OTX_v2
+  version: -1
+configuration:
+- display: Trust any certificate
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy
+  name: proxy
+  required: false
+  type: 8
+- display: Token
+  name: api_token
+  required: false
+  type: 4
+- defaultvalue: https://otx.alienvault.com
+  display: Server address
+  name: Server
+  required: false
+  type: 0
+description: Query IOCs in AlienVault
+display: AlienVault_OTX_v2
+name: AlienVault_OTX_v2
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: ip to query
+      isArray: false
+      name: ip
+      required: true
+      secret: false
+    deprecated: false
+    description: Query IP in AlienVault OTX
+    execution: false
+    name: ip
+    outputs:
+    - contextPath: IP.Address
+      description: IP address
+      type: String
+    - contextPath: IP.ASN
+      description: IP ASN
+      type: String
+    - contextPath: IP.Geo.Country
+      description: IP geographic country
+      type: String
+    - contextPath: IP.Geo.Location
+      description: IP latitude, longtitude
+      type: String
+    - contextPath: AlienVaultOTX.IP.Reputation
+      description: IP reputation
+      type: String
+    - contextPath: DBotScore.Score
+      description: The score from AlienVault
+      type: Number
+    - contextPath: DBotScore.Type
+      description: Indicator type
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: AlienVault vendor
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator we tested
+      type: String
+  - arguments:
+    - default: false
```

Comment:
should be default
```suggestion
    - default: true
```

Reply:
I changed, But I will appreciate an explanation, I will call you.
👍